### PR TITLE
New moderation tools: block & mute

### PR DESCRIPTION
**Pull request by seanyesmunt**

### CODE REVIEW

File: ui/component/commentMenuList/view.jsx [hunk 1]
```diff
@@ -123,16 +100,25 @@ function CommentMenuList(props: Props) {
...
         <MenuItem className="comment__menu-option" onSelect={handleCommentBlock}>
           <div className="menu__link">
             <Icon aria-hidden icon={ICONS.BLOCK} />
             {__('Block')}
           </div>
-          {/* {activeChannelIsCreator && (
-            <span className="comment__menu-help">Hide this channel's comments and block them from commenting.</span>
-          )} */}
+          {activeChannelIsCreator && (
+            <span className="comment__menu-help">Prevent this channel from interacting with you.</span>
```
Comment (jessopb) [inline ui/component/commentMenuList/view.jsx:110]: I18n?

File: ui/component/commentMenuList/view.jsx [hunk 2]
```diff
@@ -123,16 +100,25 @@ function CommentMenuList(props: Props) {
...
+        </MenuItem>
+      )}
+
+      {!commentIsMine && (
+        <MenuItem className="comment__menu-option" onSelect={handleCommentMute}>
+          <div className="menu__link">
+            <Icon aria-hidden icon={ICONS.MUTE} />
+            {__('Mute')}
+          </div>
+          {activeChannelIsCreator && <span className="comment__menu-help">Hide this channel for you only.</span>}
```
Comment (jessopb) [inline ui/component/commentMenuList/view.jsx:121]: i18n

File: ui/redux/actions/comments.js [hunk 3]
```diff
@@ -406,38 +413,204 @@ export function doCommentUpdate(comment_id: string, comment: string) {
...
+        let globalBlockList = [];
+        blockLists
+          .sort((a, b) => {
+            return 1;
+          })
+          .forEach((channelBlockListData) => {
+            const blockListForChannel = channelBlockListData && channelBlockListData.blocked_channels;
+            if (blockListForChannel) {
+              blockListForChannel.forEach((blockedChannel) => {
+                // REMOVE THIS
```
Comment (jessopb) [inline ui/redux/actions/comments.js:547]: remove what?

File: ui/redux/actions/comments.js [hunk 4]
```diff
@@ -406,38 +413,204 @@ export function doCommentUpdate(comment_id: string, comment: string) {
...
+        dispatch({
+          type: ACTIONS.COMMENT_MODERATION_BLOCK_LIST_COMPLETED,
+          data: {
+            blockList: globalBlockList
+              .sort((a, b) => new Date(a.blockedAt) - new Date(b.blockedAt))
+              .map((blockedChannel) => blockedChannel.channelUri),
+          },
+        });
+      })
+      .catch((err) => {
```
Comment (jessopb) [inline ui/redux/actions/comments.js:571]: do something with this later, maybe, I suppose.

File: ui/component/claimPreview/view.jsx [hunk 5]
```diff
@@ -329,12 +326,17 @@ const ClaimPreview = forwardRef<any, {}>((props: Props, ref: any) => {
                     actions
                   ) : (
                     <div className="claim-preview__primary-actions">
+                      {!isChannelUri && signingChannel && (
+                        <div className="claim-preview__channel-staked">
+                          <ChannelThumbnail uri={signingChannel.permanent_url} />
+                          {/* <ChannelStakedIndicator uri={signingChannel.permanent_url} inline /> */}
```
Comment (jessopb) [inline ui/component/claimPreview/view.jsx:332]: sup?

File: ui/redux/actions/comments.js (hunk 3 above)
Comment (seanyesmunt) [reply ui/redux/actions/comments.js:547]: guess I removed it but didn't remove the comment

**Pull request merged**